packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -28,17 +28,8 @@ contract Reserve is
   using FixidityLib for FixidityLib.Fraction;
   using Address for address payable; // prettier-ignore
 
-  struct TobinTaxCache {
-    uint128 numerator;
-    uint128 timestamp;
-  }
-
   mapping(address => bool) public isToken;
   address[] private _tokens;
-  TobinTaxCache public tobinTaxCache;
```

Comment:
Hey FYI I don't think we can remove these without a storage-level upgrade (since deleting them will shift storage slots for the mappings that follow, which are still being used). I think it'd be preferable to leave them in but add a comment explaining that they are deprecated. (Either way, def need to check this with some of the smart contract pros)